src/vip_ipykernel_launcher.py

```diff
@@ -0,0 +1,26 @@
+"""Taken and modified from
+https://github.com/ipython/ipykernel/blob/master/ipykernel_launcher.py
+
+Entry point for launching an ViP-IPython kernel.
+"""
+
+import subprocess
+import sys
+
+from vip_ipykernel import venv_search
+
+if __name__ == '__main__':
+    # Remove the CWD from sys.path while we load stuff.
+    # This is added back by InteractiveShellApp.init_path()
+    if sys.path[0] == '':
+        del sys.path[0]
+
+    args = sys.argv
```

Comment:
```suggestion
    args = sys.argv.copy()
```

Doesn't really matter here, but it's a good idea to avoid modifying globals if possible.